aboleth/losses.py

```diff
@@ -92,8 +92,8 @@ def max_posterior(log_likelihood, regulariser):
 
     .. code-block:: python
 
-        noise = tf.Variable(1.0)
-        likelihood = tf.distributions.Normal(loc=NN, scale=ab.pos(noise))
+        noise = tf.nn.softplus(tf.Variable(ab.inverse_softplus(1.0)))
```

Comment:
This comes up a lot -- I would define ab.pos_variable(x) that implemented this line. That's what I'd put in the ab namespace, rather than the inverse_softplus function (which could be local to that file), then all the demos could use it too